composition/tests/v1/resolvability.test.ts

```diff
@@ -1696,6 +1696,11 @@ describe('Field resolvability tests', () => {
       ),
     );
   });
+
+  test('that unresolvable field due to incorrect `@key` is handled correct', () => {
```

Comment:
that errors are returned for unresolvable fields involving a shared root query field and unreachable nested entities